test/e2e/utils/utils.go

```
@@ -406,3 +444,230 @@ func WaitForResourceVersionChange(ctx context.Context, client *resources.Resourc
+	u.RawQuery = q.Encode()
+
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, u.String(), nil)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create request: %w", err)
+	}
+
+	resp, err := c.httpClient.Do(req)
+	if err != nil {
+		return nil, fmt.Errorf("alertmanager request failed: %w", err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)